### ISSUE TITLE
How to run the mopAOS with IDEA?

### ISSUE THREAD

**Issue by zhuericmax** [seaklab/mopAOS#3]
I add src and example as source, and run the example 1, but it appears that method inverse wasn't defined. How can I run the project correctly?
![1](https://user-images.githubusercontent.com/47269332/70705954-f75cf200-1d0f-11ea-9465-820a52ec3a6f.png)
![2](https://user-images.githubusercontent.com/47269332/70705963-f88e1f00-1d0f-11ea-9fd8-1082a64444d7.png)

**Comment by pd385** [comment]
Hi, 
Try changing the source to 'java' in src>main. It should run.

**Comment by thathulk** [comment]
Hi, 
Thank you. As you said, I added these folders(down below)as source folders, but it didn't work. Or am i misunderstanding your point?

**Comment by zhuericmax** [comment]
Hi, thank you. As you said, I added these folders(down below)as source folders, but it didn't work. Or am i misunderstanding your point?
![image](https://user-images.githubusercontent.com/47269332/70765419-b4ddf880-1d95-11ea-9832-034869fa4052.png)

**Comment by pd385** [comment]
It looks like your your source folder is "src" instead of "java". If you change it to be just "java", it will work. Also try to run from branch "develop" instead of "master". For me, custom example defined in aos works fine. Let me know how it goes.

**Comment by zhuericmax** [comment]
Yeah, I did what you said and it worked finally. But it only works for branch 'develop'. What should I do if I want to run the branch 'master'? Because it seems like that it's much more relative to the paper.

**Comment by pd385** [comment]
I guess it depends on what you want to do. The master branch hasn't been updated in a while, and might not work. The complete code for Adaptive Operator Selection is available on the develop branch. Let me know if you have more questions.

**Comment by zhuericmax** [comment]
Thank you so much! I've got it!

**Comment by thathulk** [comment]
One more little question: Is this normal that something comes out after the results(Exception in thread "main" org.apache.commons.math3.exception.NotStrictlyPositiveException: 0 is smaller than, or equal to, the minimum (0)... ...) when I run Example1Custom?

**Comment by zhuericmax** [comment]
One more little question: Is this normal that something comes out after the results(Exception in thread "main" org.apache.commons.math3.exception.NotStrictlyPositiveException: 0 is smaller than, or equal to, the minimum (0)... ...) when I run Example1Custom?

![image](https://user-images.githubusercontent.com/47269332/71054990-f9e79d80-218e-11ea-8872-81b4d1096fe5.png)
